tsc/process/security.md

```diff
@@ -0,0 +1,100 @@
+**Security and OpenVDB**
+
+The OpenVDB project takes security very seriously.
+
+OpenVDB was not originally built as an outward-facing library.
+Users should exercise caution when working with untrusted data.
+Code injection bugs have been found in much simpler data structures,
+so it would be foolish to assume OpenVDB is immune.
+
+OpenVDB is also focused on high performance.   It will rely on
+incoming parameters being valid.  For example, array bounds checking is
+intentionally avoided.  Likewise, integer overflow concerns are
+intentionally not addressed.
+
+***Reporting***
+
+If you discover a security vulnerability that you feel cannot
+be disclosed publicly, please submit it to security@openvdb.org.
+This will go to a private mailing list of the TSC where we will
+endeavour to triage and respond to your issue in a timely manner.
+
+***Outstanding Security Issues***
+
+None
+
+***Addressed Security Issues***
+
+None
+
+***File Format Expectations***
+
+Attempting to read a .vdb file will:
+* Return success and produce a valid VDB data structure in memory
+* Fail with an error
+* Execute forever
+* Run out of memory
+
+The last two options may be surprising.  VDBs, however, are designed
+as open-ended containers of production data that may be terabytes in size.
+
+It is a bug if some file causes the library to crash.  It is a serious
+security issue if some file causes arbitrary code execution.
+
+***Runtime Library Expectations***
+
+We consider the library to run with the same privilage as the linked
```

Comment:
typo - privilage

Reply:
I had it spelled even worse in the original :>  When I fixed I obviously didn't fix far enough :>